api/v1alpha1/appgroup_types.go

```diff
@@ -155,12 +155,25 @@ type ApplicationGroupSpec struct {
 	Interval *metav1.Duration `json:"interval,omitempty"`
 }
 
+type ParentChartOrder string
+
+const (
+	Parallel ParentChartOrder = "parallel"
+	First    ParentChartOrder = "first"
+	Last     ParentChartOrder = "last"
+)
+
 // Application spec and dependency on other applications
 type Application struct {
 	// DAG contains the dependency information
```

Comment:
Just a styling thing that should have been fixed in the past, can you put newlines between each of the API values?

Reply:
Do you mean each member of the `Application` struct?

Reply:
Yes

Reply:
@jonathan-innis  I am going to fix this in a different PR. If everything looks good functionally can we get this merged in?